src/lib/utils/enhanceConfiguration.ts

```diff
@@ -1,5 +1,6 @@
-import { SearchParameters, Widget } from '../../types';
-import mergeDeep from './mergeDeep';
+import { Widget } from '../../types';
+import mergeSearchParameters from './mergeSearchParameters';
+import { SearchParameters } from 'algoliasearch-helper';
```

Comment:
looks like we will need #3940 for this to fully work

Reply:
Nope it's not linked. It's because the Helper is written in cjs (the [error](https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module) come from Rollup). To avoid the error without any Rollup configuration use:

```js
import algoliasearchHelper from 'algoliasearch-helper';

// ...

new algoliasearchHelper.SearchParameters()
```

Reply:
Good call! There’s more places than this with the destructuring pattern. Should we just move them all to the dot style or only the ones the build fails for?

Reply:
Inside the source code we have other one? It would surprise me that they don't fail. Inside the test is fine though since we don't run Rollup on them.